src/openrct2/world/TileInspector.cpp

```diff
@@ -854,7 +855,8 @@ namespace OpenRCT2::TileInspector
             if (ride == nullptr)
                 return std::make_unique<GameActions::Result>(GameActions::Status::Unknown, STR_NONE);
 
-            auto trackBlock = TrackBlocks[type];
+            const auto& teDescriptor = GetTrackElementDescriptor(type);
+            auto trackBlock = teDescriptor.Block;
```

Comment:
```suggestion
            const auto* trackBlock = teDescriptor.Block;
```
Always specify pointer/reference type on auto.